api/src/main/java/org/apache/gravitino/SupportsTags.java

```diff
@@ -17,11 +17,12 @@
  * under the License.
  */
 
-package org.apache.gravitino.tag;
+package org.apache.gravitino;
 
 import org.apache.gravitino.annotation.Evolving;
 import org.apache.gravitino.exceptions.NoSuchTagException;
 import org.apache.gravitino.exceptions.TagAlreadyAssociatedException;
+import org.apache.gravitino.tag.Tag;
```

Comment:
Why do we do such changes in the API level? It will basically break the compatibility. If there's no strong reason, we should not do a such change.

Reply:
If we do not move the Supports in the API module, it will conflict with the one in the core module. I referred to other designs under the API, such as catalog, and moved it out of the Tag folder. Would you happen to have any good suggestions?

Reply:
I think you can have two options:

1) No need to use dispatcher pattern, just implement the listener logic in `TagManager`.
2) To refer to `PatitionDispatcher`, just add the methods in this `TagDispatcher` interface, no need to have another `SupportsXXX` interface.

Reply:
Hi Jerry,
I decided to go with Option 2 and have completed the implementation.